tests/smoke_test.go

```diff
@@ -186,10 +185,12 @@ func TestLanguageConvertComponentSmoke(t *testing.T) {
 			e.CWD = filepath.Join(e.RootPath, "out")
 			e.RunCommand("pulumi", "stack", "init", "test")
 
-			// TODO[pulumi/pulumi#13075]: Skipping `up` until we have a way to tell the language host to
-			// install dependencies.
-			// e.RunCommand("pulumi", "up", "--yes")
-			// e.RunCommand("pulumi", "destroy", "--yes")
+			// TODO: This doesn't work for Go yet because the source code convert emits is not valid
```

Comment:
Do we have an issue for this that we could link?  If we don't is it worth creating one?  This seems like a place where it's easy for a TODO to stay around forever because we forget about it otherwise (even after the issue is fixed)

Reply:
No, we could raise one specifically for this though rather than relying on it getting done as part of the codegen fixes this quarter.